vitrivr-engine-module-fes/build.gradle

```diff
@@ -0,0 +1,36 @@
+
+task generateFESClient(type: org.openapitools.generator.gradle.plugin.tasks.GenerateTask) {
+    generatorName = "kotlin"
+    inputSpec = "$rootDir/vitrivr-engine-module-fes/feature-extraction-server.json".toString()
```

Comment:
Is the referenced OpenAPI specification always up-to-date? I'd rather like to see a link to a github link, e.g. to the version on the main branch of this file.

Reply:
I agree with this. Sadly, the openapi spec had to be manually edited a little bit to remove two lines that the kotlin openapi generator doesn't like. I will add the edited spec to the fes repo, documenting how it was produced.